src/goto-programs/set_properties.cpp

```diff
@@ -110,8 +112,11 @@ void set_properties(
     if(!it->second.is_inlined())
       set_properties(it->second.body, property_set);
 
-  if(!property_set.empty())
-    throw "property "+id2string(*property_set.begin())+" not found";
+  INVARIANT_WITH_DIAGNOSTICS(
+    property_set.empty(),
+    "Property ",
```

Comment:
start with lower case (several occurrences)

Reply:
This one is a message that is aimed at the user; it is NOT checking an invariant!